lgc/include/lgc/state/AbiUnlinked.h

```diff
@@ -75,6 +75,10 @@ const static char SamplePatternIdx[] = "$samplePatternIdx";
 // The value of the relocation is deviceIdx from the pipeline state.
 const static char DeviceIdx[] = "$deviceIdx";
 
+// Pushconst offset is "pushconst"
+// The value of the relocation is the offset of the pushconst reource node in the pipeline state.
+const static char Pushconst[] = "pushconst";
```

Comment:
Better to name it PushConst[] = ..
"pushconst" is accepted if you don't want "push.const" or "push_const".

Reply:
That is the name that was already in use, so I'll keep it the same.